local-running/compose.yaml

```diff
@@ -313,3 +313,26 @@ services:
       - "5012:5012"
     volumes:
       - "../../ipv-config/stubs/di-ipv-credential-issuer-stub:/etc/client-configs:ro"
+
+  bav-stub:
+    image: cri-stub:latest
+    container_name: bav-stub
+    build:
+      context: ../../ipv-stubs/di-ipv-credential-issuer-stub
+      dockerfile: Dockerfile
+    environment:
+      - CREDENTIAL_ISSUER_NAME=Bank account verification (Stub)
```

Comment:
Might be better to call this just "Bank account". Looking at the flows it's an evidence CRI